### PR TITLE
Move telemetry over to new E2 subscribe/indication channel

### PR DESCRIPTION
**Pull request by shadansari**

### CODE REVIEW

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -275,36 +276,35 @@ func GetManager() *Manager {
...
-		Type:   update.GetMessageType().String(),
+		Type:   update.GetHdr().GetMessageType().String(),
 		Object: update,
 	}
 
-	err := m.telemetryStore.Put(&update)
-	if err != nil {
-		log.Fatalf("Could not put message %v in telemetry store %s", update, err.Error())
-	}
+	/*
```
Comment (kuujo) [inline pkg/manager/manager.go:285]: Should this be here?

Comment (tomikazi) [reply pkg/manager/manager.go:285]: Good eye.

Comment (shadansari) [reply pkg/manager/manager.go:285]: Ooops - yes. I had commented while debugging something. Will push a fix.

File: pkg/manager/manager.go [hunk 2]
```diff
@@ -275,36 +276,35 @@ func GetManager() *Manager {
...
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[0].CqiHist[0],
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[0].GetEcgi().GetEcid(),
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[1].CqiHist[0],
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[1].GetEcgi().GetEcid(),
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[2].CqiHist[0],
+			msg.GetRadioMeasReportPerUE().RadioReportServCells[2].GetEcgi().GetEcid(),
 		)
 	default:
-		log.Fatalf("Telemetry update has unexpected type %T", update.MessageType)
+		log.Fatalf("Telemetry update has unexpected type %d", update.GetHdr().GetMessageType())
```
Comment (SeanCondon) [inline pkg/manager/manager.go:305]: I think the message type as a string is better than a number in the log

Comment (shadansari) [reply pkg/manager/manager.go]: fixed

**Pull request merged**